tests/unit/routes/test_patient_router.py

```diff
@@ -0,0 +1,67 @@
+"""
+unit.routes.test_patient_router.py
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module contains the unit tests for the recordlinker.routes.patient_router module.
+"""
+
+import uuid
+
+from recordlinker import models
+
+
+class TestCreatePerson:
+    def test_invalid_reference_id(self, client):
+        response = client.post("/patient/123/person")
+        assert response.status_code == 422
+
+    def test_invalid_patient(self, client):
+        response = client.post(f"/patient/{uuid.uuid4()}/person")
+        assert response.status_code == 404
+
+    def test_create_person(self, client):
+        original_person = models.Person()
+        patient = models.Patient(person=original_person, data={})
+        client.session.add(patient)
+        client.session.flush()
+
+        resp = client.post(f"/patient/{patient.reference_id}/person")
+        assert resp.status_code == 201
+        assert resp.json()["patient_reference_id"] == str(patient.reference_id)
+        assert resp.json()["person_reference_id"] != str(original_person.reference_id)
```

Comment:
Since `original_person` only had one `patient` linked to it, what happens to that person after that `patient` is removed from it?

Reply:
Yeah thats true....maybe its better to leave it for now and open another ticket later if we want to cleanup the orphaned cluster. But might be worth leaving it around just in case of mistakes like you said for now.

Reply:
It's a good thought, and appreciate you bringing it up.  I think the biggest negative to leaving it in the DB is storage space, but given how slim that table is in the first place, just two columns, it's fairly negligible.